scalafmt-tests/src/test/scala-2.12/org/scalafmt/ScalafmtProps.scala

```diff
@@ -34,14 +34,16 @@ class ScalafmtProps extends FunSuite with FormatAssertions {
           case Formatted.Success(formatted) =>
             assertFormatPreservesAst[Source](code, formatted)
             val formattedSecondTime = Scalafmt.format(formatted, config).get
-            Diffs.assertNoDiff(formattedSecondTime, formatted, diff => {
-              throw DiffFailure(
-                "Idempotency",
-                formatted,
-                formattedSecondTime,
-                diff
-              )
-            }, "Idempotency")
+            try assertNoDiff(formattedSecondTime, formatted, "Idempotency")
```

Comment:
looks like munit is not a third-party package. would it make sense to display the diff as part of `assertNoDiff` and make the change in munit?

Reply:
It will be good! This is just work around api limitations.